_overviews/cheatsheets/index.md

```diff
@@ -26,270 +26,271 @@ permalink: /cheatsheets/index.html
       <td> </td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>var x = 5</code></pre><br /> <span class="label success">Good</span><br><pre class="highlight"><code>x=6</code></pre></td>
-      <td>variable</td>
+      <td><pre class="highlight"><code>var x = 5</code></pre><br /> <span class="label success">Good</span><br><pre class="highlight"><code>x = 6</code></pre></td>
+      <td>Variable.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>val x = 5</code></pre><br /> <span class="label important">Bad</span><br><pre class="highlight"><code>x=6</code></pre></td>
-      <td>constant</td>
+      <td><pre class="highlight"><code>val x = 5</code></pre><br /> <span class="label important">Bad</span><br><pre class="highlight"><code>x = 6</code></pre></td>
+      <td>Constant.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>var x: Double = 5</code></pre></td>
-      <td>explicit type</td>
+      <td>Explicit type.</td>
     </tr>
     <tr>
       <td><span id="functions" class="h2">functions</span></td>
       <td> </td>
     </tr>
     <tr>
       <td><span class="label success">Good</span><br> <pre class="highlight"><code>def f(x: Int) = { x * x }</code></pre><br /> <span class="label important">Bad</span><br> <pre class="highlight"><code>def f(x: Int)   { x * x }</code></pre></td>
-      <td>define function <br /> hidden error: without = it’s a Unit-returning procedure; causes havoc</td>
+      <td>Define function.<br />Hidden error: without <code>=</code> it’s a <code>Unit</code>-returning procedure; causes havoc.</td>
```

Comment:
The site's CSS for `<code>` is adding weird horizontal whitespace so it looks strange in some of these places you added it, e.g. "`Unit `-recording".

Reply:
Agree, this one looked bad, fixed it and overall the whitespace makes it a bit weird, but I still think it is better to use `<code>`.

Reply:
I agree.  It should be possible to use `<code>`.  At some point we need to fix the CSS for `<code>`.